code/modules/power/smes.dm

```diff
@@ -435,6 +485,12 @@
 	charge = INFINITY
 	..()
 
+/obj/machinery/power/smes/proc/emp_end() //used to check if SMES was EMPED and reset the overlay state
+	is_emped = FALSE
+	update_icon()
+	log_smes()
+
```

Comment:
one less newline please, also what is this log FOR if being emp'd isn't a part of its readout